app/controllers/school_controller.py

```diff
@@ -0,0 +1,11 @@
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.get("/health", status_code=200)
+
+async def info():
```

Comment:
It is a good practice to name the controller method like a resource name. renamed to health